Common/src/Pollster/RequestProcessor.cs

```diff
@@ -37,205 +37,61 @@
 {
-  private readonly ActivitySource        activitySource_;
-  private readonly ILogger               logger_;
-  private readonly IObjectStorage        resourcesStorage_;
-  private readonly ISubmitter            submitter_;
-  private readonly IWorkerStreamHandler  workerStreamHandler_;
+  private readonly ActivitySource       activitySource_;
+  private readonly ILogger              logger_;
+  private readonly IObjectStorage       resourcesStorage_;
+  private readonly ISubmitter           submitter_;
+  private readonly IWorkerStreamHandler workerStreamHandler_;
 
   public RequestProcessor(IWorkerStreamHandler  workerStreamHandler,
                           IObjectStorageFactory objectStorageFactory,
                           ILogger               logger,
                           ISubmitter            submitter,
                           ActivitySource        activitySource)
   {
-    workerStreamHandler_  = workerStreamHandler;
-    logger_               = logger;
-    submitter_            = submitter;
-    activitySource_       = activitySource;
-    resourcesStorage_     = objectStorageFactory.CreateResourcesStorage();
+    workerStreamHandler_ = workerStreamHandler;
+    logger_              = logger;
+    submitter_           = submitter;
+    activitySource_      = activitySource;
+    resourcesStorage_    = objectStorageFactory.CreateResourcesStorage();
   }
 
   public async Task<Task> ProcessAsync(IQueueMessageHandler  messageHandler,
                                        TaskData              taskData,
                                        Queue<ComputeRequest> computeRequests,
                                        CancellationToken     cancellationToken)
   {
-    try
-    {
-      var result = await ProcessInternalsAsync(taskData,
-                                               computeRequests,
-                                               cancellationToken)
-                     .ConfigureAwait(false);
-
-      messageHandler.Status = QueueMessageStatus.Processed;
-      return result;
-    }
-    catch (Exception e)
-    {
-      logger_.LogError(e,
-                       "Error while processing request");
-      // TODO cancel session dispatch, cancel task ?
-      //await submitter_.CancelDispatchSessionAsync(taskData.SessionId,
-      //                                            dispatch.Id,
-      //                                            cancellationToken)
-      //                .ConfigureAwait(false);
-
-      if (!await HandleExceptionAsync(e,
-                                      taskData,
-                                      messageHandler,
-                                      cancellationToken)
-             .ConfigureAwait(false))
-      {
-        throw;
-      }
-
-      throw new ArmoniKException("An error occurred while executing. Error has been managed.");
-    }
-  }
-
-
-  [PublicAPI]
-  private async Task<bool> HandleExceptionAsync(Exception            e,
```

Comment:
Could you add in the PR description the changes in the code logic that allows this deletion

Reply:
done